main.css

```diff
@@ -19,6 +19,31 @@ ul {
   padding: 0;
 }
 
+body {
+  position: relative;
+}
+
+/* blur is for header and main for popup */
+.blur {
+  filter: blur(0.8rem);
+  background-color: rgba(0, 0, 0, 0.5);
+}
+
+/* disable-scroll for body for popup */
+.disable-scroll {
```

Comment:
- [ ] On mobile the popup is not completed and can't scroll down, kindly fix this or remove the ```.disable-scroll``` so the user can reach the popup links

![Screenshot_20220512_210557](https://user-images.githubusercontent.com/25227022/168159363-2204ea15-01ed-44e2-8974-2c66b701f015.png)